### PR TITLE
Drop Python 2.7 and 3.5 from Travis

### PR DESCRIPTION
**Pull request by rhysmeister**
##### SUMMARY
Drop Python 2.7 and 3.5 from the Travis tests

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
.travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 language: python
 python:
-  - "2.7"
   - "3.5"
```
Comment (cognifloyd) [inline .travis.yml:4]:
```suggestion
```
Can we drop 3.5 as well?

**Review by cognifloyd — COMMENTED** [review]
Thanks!
I meant to do this, but I haven't had much time lately as there is a lot to learn at my new job :)

What do you think of dropping 3.5 here as well? I think only one version of python should be enough to test the roles, as nothing is particularly python version dependent in the roles themselves (at least for the controller-side).

File: .travis.yml (hunk 1 above)
Comment (rhysmeister) [reply .travis.yml:4]: Let's do it. Probably doesn't add much value anyway.

**Comment by rhysmeister** [comment]
Travis tests have all passed. Merging.

**Pull request merged**